harmony-one/harmony
# node.sh v3.0

This PR re-write the node.sh.

The new node.sh greatly simplified the node.sh.
The code is mainly borrowed from prysm.sh (eth2 client).

The new node.sh is only used to

* download latest harmony binary or specified version
* download from github release page
* use harmony public key to do verify the pgp signature of the binary

`539910B02B5BAC637F6615C9799ACE3A9B83DF60` is the fingerprint of the harmony key.

To sign the binary, using the following command,
```
gpg -u 539910B02B5BAC637F6615C9799ACE3A9B83DF60 --output harmony-amd64.sig --detach-sig harmony-amd64
```

To verify the key, using the following command,
```
gpg -u 539910B02B5BAC637F6615C9799ACE3A9B83DF60 --verify harmony-amd64.sig harmony-amd64
```
